inspec.yml

```diff
@@ -0,0 +1,414 @@
+    
+#SV-230374  
+  - name: emergency_account_expire_time
+    desc: Emergency (short term) should be set with an expiration date/time
+    type: Hash
+    <% if ENV['BASELINE'] == 'High-HVA' || ENV['BASELINE'] == 'Moderate-HVA' %>
+    value: 
+      hours: 12
+      days: 0.5
+    <% elsif ENV['BASELINE'] == 'High' || ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'].nil?  %>
+    value: 
+      hours: 24
+      days: 1
+    <% end %>   
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+#SV-230483, SV-244543
+  - name: storage_volume
+    desc: Amount of audit storage volume
+    type: Numeric
+    value: 80
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: storage_space_left
+    desc: Amount of audit storage volume left
+    type: Numeric
+    value: 20
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+#SV-230332, SV-230333, SV-230334, SV-230335, SV-230336, SV-230337, SV-230338, SV-230340, SV-230341, SV-230344, SV-230345
+  - name: unsuccessful_attempts
+    desc: Maximum number of unsuccessful attempts before lockout
+    type: Numeric
+    <% if ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'] == 'Moderate-HVA' || ENV['BASELINE'] == 'Low' || ENV['BASELINE'] == 'Low-HVA' || ENV['BASELINE'].nil? %>
+    value: 5
+    <% else %>
+    value: 3
+    <% end %>
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: fail_interval 
+    desc: Interval of time in which the consecutive failed logon attempts must occur in order for the account to be locked out (time in seconds)
+    type: Numeric
+    value: 7200
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+  - name: lockout_time
+    desc: Minimum amount of time account must be locked out after failed logins. This attribute should never be set greater than 604800 (time in seconds). ("0" or "never" locks until released by an administrator)
+    type: Numeric
+    <% if ENV['BASELINE'] == 'High' || ENV['BASELINE'] == 'High-HVA' %>
+    value: 0
+    <% elsif ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'] == 'Moderate-HVA' || ENV['BASELINE'].nil? %>
+    value: 3600
+    <% else %>
+    value: 900
+    <% end %>
+    profile: redhat-enterprise-linux-8-stig-baseline
+
+#SV-230363
+  - name: difok
+    desc: Minimum number of characters that must be different from previous password
+    type: Numeric
+    <% if  ENV['BASELINE'] == 'Moderate' ||  ENV['BASELINE'] == 'High' ||  ENV['BASELINE'] == 'Low' || ENV['BASELINE'].nil?%>
+    value: 12
+    <% else %>
+    value: 15
+    <% end %>
+    profile: redhat-enterprise-linux-8-stig-baseline 
+
+#SV-230368
+  - name: min_reuse_generations
+    desc: Number of reuse generations
+    type: Numeric
+    <% if ENV['BASELINE'] == 'High' %>
+    value: 12
+    <% elsif ENV['BASELINE'].nil? || ENV['BASELINE'] == 'Moderate' || ENV['BASELINE'] == 'Low' %>
+    value: 6
+    <% else %>
+    value: 0
+    <% end %>
+    profile: redhat-enterprise-linux-8-stig-baseline
```

Comment:
For HVA systems, ARS says `PasswordHistorySize = reuse is disallowed;`. Currently, `0` is set for HVA which would mean that they can reuse passwords easily. I'm unable to find the max value allowed for `pam_pwhistory.so remember=<value>`, but it should be at least 12 imo.

Reply:
Done https://github.com/CMSgov/cms-ars-5.0-red-hat-enterprise-linux-8-stig-overlay/pull/1/commits/9963d0a80dd3dd91c550f0804494d7c1fa6232bc